src/gui/gui_common_elements.hpp

```diff
@@ -1232,32 +1100,42 @@ class nation_ships_text : public standard_nation_text {
 		return total;
 	}
 
-	public:
+public:
 	std::string get_text(sys::state& state, dcon::nation_id nation_id) noexcept override {
 		return std::to_string(get_ship_count(state, nation_id));
 	}
 };
 
 class nation_brigades_text : public standard_nation_text {
```

Comment:
you have a class here that claims to contain the number of brigades but instead counts the number of armies. I think this is likely to be confusing

Reply:
ill rename it to armies, and likewise for nation_ships_text into navies

Reply:
also, removed nation_ships_text, it seems nation_navies_text deprecated it